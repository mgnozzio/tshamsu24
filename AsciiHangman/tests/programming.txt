programming
p
r
o
g
a
m
i
n
