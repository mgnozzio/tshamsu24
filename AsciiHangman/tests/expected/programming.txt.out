Welcome to the ASCII Version of Hangman!
For best results, please run this from the command line.
Please enter a secret word: 
















































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































   ____  
  |    | 
  |      
  |      
  |      
__|__    
Puzzle to solve: _ _ _ _ _ _ _ _ _ _ _ 
Please guess a letter: P
Puzzle to solve: P _ _ _ _ _ _ _ _ _ _ 
Please guess a letter: R
Puzzle to solve: P R _ _ R _ _ _ _ _ _ 
Please guess a letter: O
Puzzle to solve: P R O _ R _ _ _ _ _ _ 
Please guess a letter: G
Puzzle to solve: P R O G R _ _ _ _ _ G 
Please guess a letter: A
Puzzle to solve: P R O G R A _ _ _ _ G 
Please guess a letter: M
Puzzle to solve: P R O G R A M M _ _ G 
Please guess a letter: I
Puzzle to solve: P R O G R A M M I _ G 
Please guess a letter: N
Success!  Player 2 wins!
